mfkdf2/src/derive/factors/hmacsha1.rs

```diff
@@ -65,7 +64,6 @@ pub fn hmacsha1(response: HmacSha1Response) -> MFKDF2Result<MFKDF2Factor> {
       params:        None,
       padded_secret: [0u8; 32].to_vec(),
```

Comment:
why does this need to be in here if it's just zeros?

Reply:
Okay, I see. It may be better to swap to `MaybeUninit` in the future. Can you take note of this?